### PR TITLE
fix: test bug due to bad mocks

### PR DESCRIPTION
**Pull request by JGSweets**
Fixes an issue with non-forked tests when loading the tf_model.

Now returns a callable object instead to trigger the callable command.

### CODE REVIEW

File: dataprofiler/tests/labelers/test_character_level_cnn_model.py [hunk 1]
```diff
@@ -35,11 +35,12 @@
 }
 
 
-def mock_open(filename, *args):
+def mock_open(filename, *args, **kwargs):
     if filename.find("model_parameters") >= 0:
         return StringIO(json.dumps(mock_model_parameters))
     elif filename.find("label_mapping") >= 0:
         return StringIO(json.dumps(mock_label_mapping))
+    return StringIO()
```
Comment (JGSweets) [inline dataprofiler/tests/labelers/test_character_level_cnn_model.py:43]: extra conditions to allow for debugging with IPDB. I think should stay for future cases if debugging is needed.

File: dataprofiler/tests/labelers/test_character_level_cnn_model.py [hunk 2]
```diff
@@ -391,10 +392,7 @@ def test_threshold_layer(self):
         self.assertCountEqual(threshold_output, expected_threshold_output)
 
     @mock.patch("tensorflow.keras.Model.save", return_value=None)
-    @mock.patch("tensorflow.keras.models.load_model", return_value=None)
-    @mock.patch(
-        "dataprofiler.labelers.character_level_cnn_model.callable", return_value=True
-    )
+    @mock.patch("tensorflow.keras.models.load_model", return_value=mock.Mock())
```
Comment (JGSweets) [inline dataprofiler/tests/labelers/test_character_level_cnn_model.py:395]: fix #1.

File: dataprofiler/tests/labelers/test_character_level_cnn_model.py [hunk 3]
```diff
@@ -430,11 +428,8 @@ def test_save(self, mock_open, *mocks):
         StringIO.close(mock_file)
 
     @mock.patch("tensorflow.keras.Model.save", return_value=None)
-    @mock.patch("tensorflow.keras.models.load_model", return_value=None)
+    @mock.patch("tensorflow.keras.models.load_model", return_value=mock.Mock())
     @mock.patch("builtins.open", side_effect=mock_open)
-    @mock.patch(
-        "dataprofiler.labelers.character_level_cnn_model.callable", return_value=True
-    )
```
Comment (JGSweets) [inline dataprofiler/tests/labelers/test_character_level_cnn_model.py:437]: mirrored fix # 2 where the actual bug occurred

**Review by taylorfturner — APPROVED** [review]

File: dataprofiler/tests/labelers/test_character_level_cnn_model.py (hunk 3 above)
Comment (tyfarnan) [reply dataprofiler/tests/labelers/test_character_level_cnn_model.py:437]: nice

**Review by tyfarnan — APPROVED** [review]

**Pull request merged**